source/openapi3/split.yaml

```diff
@@ -2731,16 +2489,21 @@ paths:
     post:
       tags:
         - Payouts
-      summary: Retry a credit/debit Payout
+      summary: Retry a Prefailed Payment Request [DEPRECATED]
       description: |
-        Split will prefail a debit and its associated credit transaction before ever sending it to the bank if we detect a high probability of insufficient funds.
+        When debitting a Contact with an active bank connection, Split will first ensure that there are sufficient funds available before submitting the debit to the banks.
+        If a high probability of insufficient funds is detected, Split will not submit the transaction to the banks and will mark the transaction as <strong>Prefailed</strong>.
+
+        This endpoint allows you to retry the Payment Request without having to create a brand new transaction.
 
-        This endpoint allows you to retry the payout without having to recreate the parent request. e.g A Payment or Payment Request.
+        <aside class="notice">
+          Rather than using this endpoint, we now recommend creating a brand new transaction as this simplifies and maintains data integrity for the lifecycle of each attempt
+        </aside>
       operationId: RetryAPayout
       parameters:
         - name: ref
           in: path
-          description: Payout debit or credit reference
+          description: Payment Request debit or credit reference number e.g C.2 / D.72
```

Comment:
Shall we use bigger numbers to give a more accurate indication of what refs will look like? 🤷

Reply:
Absolutely, and will address this when we modernize the dates  :)